app/preload.js

```diff
@@ -33,6 +35,15 @@ function getArtworkURL() {
   return null
 }
 
+function getLikeStatus() {
+  const liked = document.querySelector('.sc-button-like.playbackSoundBadge__like.sc-button-selected')
```

Comment:
Do you think `const selectedLikeButton` would better reflect the contents of this variable?